liqo-peering/values.yaml

```diff
@@ -0,0 +1,20 @@
+role: consumer  # or provider
+
+clusterId: local
+privateWireguardKey: # placeholder for private wireguard key
+publicWireguardKey: # placeholder for public wireguard key
+
+remoteCluster: 
+  clusterId: remote
+  userCommonName: "user@cluster"  # The CN of the user in the certificate
+  podCidr: # placeholder for pod CIDR
+  publicWireguardKey: # placeholder for public wireguard key
+  kubeconfig: |
+    # Here, paste your kubeconfig YAML content in base64 encoded!!!
+    apiVersion: v1
+    clusters:
+    ... ect
```

Comment:
Can we not have this as a fake value, but just add one line of comment? E.g. # kubeconfig, base64 encoded